components/dashboard/src/admin/UserDetail.tsx

```diff
@@ -328,6 +328,25 @@ export default function UserDetail(p: { user: User }) {
     );
 }
 
+function renderBillingModeProperty(billingMode?: BillingMode): JSX.Element {
+    const text = billingMode?.mode || "---";
+
+    let details = "";
+    if (billingMode?.mode === "chargebee" && !!billingMode.teamNames && !billingMode.canUpgradeToUBB) {
+        details = `UBP blocked by: ${billingMode.teamNames.join(", ")}`;
```

Comment:
**suggestion**: Since the current approach does not scale well and relies on browser tooltips to surface the necessary information, what do you think of re-using the changes dropdown from the workspaces list page here which is a known interaction pattern for users?

This way the UX will a) scale better, b) use be more concise language, and c) provide space to clarify what's happening using the dropdown header which is more accessible.

| Changes | BEFORE | AFTER (Collapsed) | AFTER (Expanded) |
|-|-|-|-|
| <img width="371" alt="Screenshot 2022-10-27 at 2 03 42 PM" src="https://user-images.githubusercontent.com/120486/198269459-8f1fc4fe-04f6-4491-be50-f444536b0f9f.png">|  <img width="757" alt="blocked-before" src="https://user-images.githubusercontent.com/120486/198269084-eb2e0d64-39a7-4d0d-93d7-276198306655.png"> | <img width="677" alt="dropdown-collapsed" src="https://user-images.githubusercontent.com/120486/198269120-3cb588b6-c34a-4c7c-afc1-d023021204d2.png"> | <img width="677" alt="dropdown-expanded" src="https://user-images.githubusercontent.com/120486/198269106-a7d47be3-2d2b-4696-aa1a-d3f48798fe0d.png"> |

Reply:
Fixed in https://github.com/gitpod-io/gitpod/pull/12805/commits/72ea0ee2061006901217b4c6c582a39c2ef98eb8.

Reply:
Woohoo! Looking forward to see this in action.